moderation/src/main/java/pl/fratik/moderation/commands/RolaCommand.java

```diff
@@ -53,11 +59,26 @@ public boolean execute(@NotNull CommandContext context) {
         }
         Role rola = (Role) context.getArgs()[0];
         GuildConfig gc = guildDao.get(context.getGuild());
+
         if (!gc.getUzytkownicyMogaNadacSobieTeRange().contains(rola.getId())) {
             context.send(context.getTranslated("rola.not.defined"));
             return false;
         }
         try {
+            Integer maxRolesSize = gc.getMaxRoliDoZamododania();
+            if (maxRolesSize == null) maxRolesSize = 100;
```

Comment:
jeżeli w [GuildConfigu#L81](https://github.com/fratik/FratikB0T/pull/7/files#diff-3c5ea3260cf456969161072fa6f96335R81) dałeś 10, to tu też daj